Kitodo/src/main/java/org/kitodo/production/forms/TemplateBaseForm.java

```diff
@@ -15,21 +15,21 @@
 import org.apache.logging.log4j.Logger;
 import org.kitodo.data.database.beans.BaseBean;
 import org.kitodo.data.database.beans.Task;
-import org.kitodo.data.exceptions.DataException;
+import org.kitodo.data.database.exceptions.DAOException;
 import org.kitodo.production.enums.ObjectType;
 import org.kitodo.production.helper.Helper;
 import org.kitodo.production.services.ServiceManager;
-import org.kitodo.production.services.data.base.SearchDatabaseService;
+import org.kitodo.production.services.data.BaseBeanService;
 
 class TemplateBaseForm extends ProcessListBaseView {
 
     private static final Logger logger = LogManager.getLogger(TemplateBaseForm.class);
 
-    void saveTask(Task task, BaseBean baseBean, String message, SearchDatabaseService searchDatabaseService) {
+    void saveTask(Task task, BaseBean baseBean, String message, BaseBeanService baseBeanService) {
```

Comment:
These few lines don't even justify the existence of this additional intermediate abstract class. It would seem sensible to me to break up the this class and duplicate the code in both places. Then it would also become obvious that the redirect is also triggered if an error occurs when saving. (Is this really intended?) Why `evict()` is called here (it is its rarely ever used in the entire application) is also not really clear to me.

In this pull request, however, I will only remove the dead parameter.